server/api/tls/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @fileoverview API Route
+ * Route for TLS related queries
+ *
+ * Here are all routes for BigQuery TLS related queries
+ */
+
+import {BigQuery} from '@google-cloud/bigquery';
+import {Router as router} from 'express';
+import * as queries from './queries.json';
+
+const tlsApi = router();
+const bigqueryClient = new BigQuery();
+
+// TODO (SofiaVega) delete this (this route is the same as the one below)
```

Comment:
Did you mean to complete this TODO as part of this PR?

Reply:
> Cool! Have a few comments, but also:
> 
> * Perhaps update the commit message to be more descriptive? Also, is this PR meant to have both TLS and also mixed content charts? If so, commit message should reflect that.
> * Are you planning to add tests for this code?

Yes, I already have some tests, should I add them to this PR?